src/c++/uds/userLinux/uds/linux/blkdev.h

```diff
@@ -92,4 +98,20 @@ static inline loff_t bdev_nr_bytes(struct block_device *bdev)
 	return bdev->size;
 }
 
+/**
+ * blkdev_issue_zeroout - zero-fill a block range
+ * @bdev:	blockdev to write
+ * @sector:	start sector
+ * @nr_sects:	number of sectors to write
+ * @gfp_mask:	memory allocation flags (for bio_alloc)
+ * @flags:	controls detailed behavior
+ *
+ * Description:
+ *  Zero-fill a block range, either using hardware offload or by explicitly
+ *  writing zeroes to the device.  See __blkdev_issue_zeroout() for the
+ *  valid values for %flags.
+ */
+int blkdev_issue_zeroout(struct block_device *bdev, sector_t sector,
+			 sector_t nr_sects, gfp_t gfp_mask, unsigned flags);
```

Comment:
Okay, if you're going to add a signature for this function, also add a userspace implementation. If something is UDS tried to call this function, it would not work. Either define in in the header, or if it's too long, add a uds/userLinux/uds/blkdev.c file to put the implementation in.

Reply:
There is an implementation. I pointed out in the main commit message that the implementation is in the second commit.

Reply:
I've filled in the implementation but left the file in vdo/tests for now till we decide what to do. As I mentioned I was just following other code like testBIO that was already there.